mailbox/api/src/test/java/org/apache/james/mailbox/model/QuotaRootTest.java

```diff
@@ -28,4 +33,23 @@
     void shouldMatchBeanContract() {
         EqualsVerifier.forClass(QuotaRoot.class).verify();
     }
+
+    @Test
+    void asStringShouldReturnValueWhenNoDomain() {
+        String value = "#private&bob";
+        QuotaRoot quotaRoot = QuotaRoot.quotaRoot(value, Optional.empty());
+
+        assertThat(quotaRoot.asString()).isEqualTo(value);
+    }
+
+    @Test
+    void asStringShouldReturnValueWithDomainWhenHasDomain() {
+        String value = "#private&bob";
+        Domain domain = Domain.of("apache.org");
+        QuotaRoot quotaRoot = QuotaRoot.quotaRoot(value, Optional.of(domain));
+
+        String expectedValue = value + "@" + domain.asString();
```

Comment:
please don't use operation to compose expected result ... you should hardcode it in this case